sonarqube-scan/action.yml

```diff
@@ -0,0 +1,42 @@
+name: 'SonarQube Quality Scan'
+description: 'Runs various quality and security checks against changes'
+inputs:
+  FLIPPCIRCLECIPULLER_REPO_TOKEN:
+    description: secret value
+    required: true
+  SONAR_HOST_URL:
+    description: secret value
+    required: true
+  SONAR_TOKEN:
+    description: secret value
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Setup private modules access
+      run: git config --global url."https://${{ inputs.FLIPPCIRCLECIPULLER_REPO_TOKEN }}:x-oauth-basic@github.com/wishabi".insteadOf "https://github.com/wishabi"
+      shell: bash
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.19'
+        check-latest: true
+        cache: true
```

Comment:
Is this needed for sq? have you tested without?

Reply:
if we extract this (if its not needed within this job), we may be able to reuse this action across different languages